.gitignore

```
@@ -0,0 +1,4 @@
+node_modules
+npm-debug.log
+.env
+.DS_Store
```

```suggestion
.DS_Store
cache
build
```
We usually don't commit `cache` and `build` directories.